src/index.js

```diff
@@ -159,9 +159,9 @@ export async function highlightElement(elm, lang = elm.className.match(/shj-lang
  * @param {ShjOptions} [opt={}] Customization options
  */
 export let highlightAll = async (opt) =>
-	document
-		.querySelectorAll('[class*="shj-lang-"]')
-		.forEach(elm => highlightElement(elm, undefined, undefined, opt))
+	Promise.all(
```

Comment:
I think Promise.all should be awaited

Reply:
added await, however async returns a promise by default and arrow function in this form returns last value, so promise was going to be returned anyways... but it's fine both work